pkg/build/strategies/dockerfile/dockerfile_test.go

```diff
@@ -80,27 +77,22 @@ func TestGetImageScriptsDir(t *testing.T) {
 					constants.DeprecatedScriptsURLLabel: "image:///usr/other/dir",
 				},
 			},
-			ExpectedDir:   "/usr/some/dir",
-			HasAllScripts: true,
+			ExpectedDir: "/usr/some/dir",
 		},
 		{
 			Config: &api.Config{
 				BuilderImageLabels: map[string]string{
 					constants.DeprecatedScriptsURLLabel: "image:///usr/some/dir",
 				},
 			},
-			ExpectedDir:   "/usr/some/dir",
-			HasAllScripts: true,
+			ExpectedDir: "/usr/some/dir",
 		},
 	}
 	for _, tc := range cases {
-		output, hasScripts := getImageScriptsDir(tc.Config)
+		output, _ := getImageScriptsDir(tc.Config, &(Dockerfile{}))
```

Comment:
Shouldn't some of these tests check for the returned map and it's contents (or lack thereof) since the `hasScripts` bool is being changed?

Reply:
The tests here were pretty deficient from what I saw. It only tested a "true" condition and empty map without providing any file-based overrides. 

I could revise this to plop &(Dockerfile{}) in the stanzas like the "true" previously; however, it would just confuse reviewers again next time that there is an actual validation here (versus it being static/empty in all cases). I think this representation more accurately matches the current and previous value of what the test was providing us.

I looked briefly at how to mock filesystem entries but I think I'll make a bigger mess before I provide more value.